java/arcs/sdk/examples/unit_testing/ComputePeopleStats.kt

```diff
@@ -0,0 +1,27 @@
+package arcs.sdk.examples.unit_testing
+
+import arcs.core.storage.api.Handle
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+typealias Person = ComputePeopleStats_People
+typealias Stats = ComputePeopleStats_Stats
+
+/**
+ * Particle computing a median age of input people - an example for unit testing.
+ */
+class ComputePeopleStats(private val scope: CoroutineScope) : AbstractComputePeopleStats() {
+
+    override suspend fun onHandleSync(handle: Handle, allSynced: Boolean) {
+        if (!allSynced) return
+        calculateMedian(handles.people.fetchAll())
+        handles.people.onUpdate { scope.launch { calculateMedian(it) } }
+    }
+
+    suspend fun calculateMedian(people: Set<Person>) = when {
+        people.isEmpty() -> handles.stats.clear()
+        else -> handles.stats.store(Stats(people.toList().map { it.age }.sorted().let {
+            (it[it.size / 2] + it[(it.size - 1) / 2]) / 2 }
+        ))
```

Comment:
`toList` isn't needed, as `map` is available on the set itself. Also, with the chain of calls, it makes sense to wrap a bit more judiciously.


```suggestion
        else -> handles.stats.store(
            Stats(
                people.map { it.age }.sorted().let {
                    (it[it.size / 2] + it[(it.size - 1) / 2]) / 2 
                }
            )
        )
```

Reply:
Done